server/controllers/chartsController/index.js

```diff
@@ -81,17 +81,18 @@ const chartsController = {
   },
   show: async (req, res) => {
     try {
-      const { dataDb } = req.app.locals
+      const { dataDb, appDb } = req.app.locals
       const { chart_id } = req.params
-      const { userAccess } = req.session
+      const user = await UserModel.findOne(appDb, { uid: req.user.uid })
```

Comment:
Can we get the user from the request? We just fetched it in the middleware so the data will be fresh.

Reply:
[46b2027](https://github.com/AMP-SCZ/dpdash/pull/589/commits/46b20279d244f8f9e24445b14486dfc8852ae61f) 👍